types/k6/experimental/webcrypto.d.ts

```diff
@@ -395,3 +395,16 @@ export type TypedArray =
     | Uint16Array
     | Int32Array
     | Uint32Array;
+
+/**
+ * JSON Web Key Value.
+ * JWKs are not supported for now, since webcrypto doesn't support exporting key/pairs
```

Comment:
I don't fully get this sentence 🤔

Reply:
>I don't fully get this sentence

It's an attempt to mention that we have yet to support multiply (JWK**s**) since there is no functionality in our webcrypto implementation to export/import key pairs.